must_triage/inspectors/ocp.py

```diff
@@ -78,3 +78,24 @@ def pod_ready(obj):
                 f"Container '{cs['name']}' in "
                 f"pod '{obj['metadata']['name']}' is not ready")
         return result
+    @staticmethod
+    def restart_count(obj):
+        result = list()
+        status = obj['status']
+        try:
+            container_statuses = status['containerStatuses']
+        except KeyError:
+            return [dict(
+                pod_name=obj['metadata']['name'],
+                status=status,
+                )]
+        not_ready = filter(lambda cs: cs['restartCount'] != '0', container_statuses)
+        for cs in not_ready:
+            ts = cs['state'].get('terminated', dict())
+            if (ts.get('restartCount', '').lower() == 'completed' and
```

Comment:
I think `restartCount` is part of the `cs` object, isn't it? Also, its value will be an integer, not a string.